client/packages/lowcoder/src/i18n/locales/en.ts

```diff
@@ -2092,12 +2092,12 @@ export const en = {
     aboutUs: "",
     changeLog: "",
     introVideo: "",
-    devNpmPlugin: "https://docs.lowcoder.dev/build-plugins/develop-your-first-plugin",
+    devNpmPlugin: "https://docs.lowcoder.cloud/build-plugins/develop-your-first-plugin",
     devNpmPluginText: "How to develop npm plugin",
-    useHost: "https://docs.lowcoder.dev/self-hosting/access-local-database-or-api",
+    useHost: "https://docs.lowcoder.cloud/self-hosting/access-local-database-or-api",
     eventHandlerSlowdown:
-      "https://docs.lowcoder.dev/build-apps/event-handlers#debounce-and-throttle",
-    thirdLib: "https://docs.lowcoder.dev/build-apps/write-javascript/use-third-party-libraries",
+      "https://docs.lowcoder.cloud/build-apps/event-handlers#debounce-and-throttle",
+    thirdLib: "https://docs.lowcoder.cloud/build-apps/write-javascript/use-third-party-libraries",
```

Comment:
https://docs.lowcoder.cloud/lowcoder-extension/use-third-party-libraries-in-apps